lib/ci/ci_failure_parser.js

```diff
@@ -155,7 +155,7 @@ const FAILURE_FILTERS = [{
 }, {
   // TODO: match indentation to avoid skipping context with '...'
   filter(ctx, text) {
-    const pattern = /not ok \d+[\s\S]+? {2}\.\.\.\r?\n/mg;
+    const pattern = /^not ok \d+[\s\S]+? {2}\.\.\.\r?\n/mg;
```

Comment:
Following the same logic
```suggestion
    const pattern = /^not ok \d+[\s\S]+? {2}\.\.\.$/mg;
```

Reply:
thanks

Reply:
Changing `\r?\n` to `$` will make this regex not match `\r?\n` after `...`, this is why the test fails. May I revert this change?

Reply:
Reason to revert:

1. after the change, the `reason` of the parse result has `\n` at the end of each line except the last line `...`. I thinks we would better keep consistent: add `\n` to the last line also.
2. avoid potential breaking change, if anywhere consume the reason property.